src/Features/SignUp/SignUpContainer.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+
+import { Card, Grid, Typography } from '@material-ui/core';
+import { useDispatch } from 'react-redux';
+
+import Button from '../../components/Button/Button';
+import TextField from '../../components/TextField/TextField';
+import { signUp } from './actions';
+
+function SignUpContainer() {
+  const dispatch = useDispatch();
+  const [email, setEmail] = useState();
+  const [password, setPassword] = useState();
+
+  const signUpUser = async () => {
+    dispatch(signUp({ email, password }));
+  };
+  return (
+    <Grid alignItems="center" container direction="column" justifyContent="center" style={{ minHeight: '100vh' }}>
+      <Typography variant="h1">Sign Up</Typography>
+      <br />
+      <Card margin="auto" maxWidth="400px" minWidth="300px">
+        <div>
+          <TextField label="Enter Email" onChange={(e) => setEmail(e.target.value)} type="email" variant="outlined" />
```

Comment:
cant we define handleChange function instead of an anonymous function? Ans use handleChange for both input field